src/components/articles/ArticlesSearch.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Card } from 'react-bootstrap';
+import parse from 'html-react-parser';
+import '../../assets/styles/searchComponent.scss';
+import '../../assets/styles/profile.scss';
+
+class ArticlesSearch extends Component {
+  render() {
+    const { value } = this.props;
+    const { slug, title, updated_at, readtime, image, url, author, body } = value;
+    const options = { month: 'long', day: 'numeric', year: 'numeric' };
+    const date_format = new Date(updated_at).toLocaleString('en-US', options);
+    return (
+      <Card className="url">
+        <div className="article">
+          <Link to={`articles/${slug}`}>
+            <div className="container-fluid">
+              <div className="row">
+                <div className={image === '' ? 'hidden ' : 'col-sm-2 col-md-4'}>
+                  <Card.Img src={image} alt="article-image" className="article-image" />
+                </div>
+                <div className={image === '' ? 'col-sm-12 col-md-12' : 'col-sm-4 col-md-8'}>
+                  <div className="col-md-12 col-xs-12">
+                    <div className="media">
+                      <div className="row">
+                        <div className="col-sm-3">
+                          <img
+                            src={require('../../assets/images/avatar.png')}
```

Comment:
I suggest that you have it this way:
`
import avatar from '../../assets/images/avatar.png'`

so that you use the imported image as below.

```
<img src={avatar} alt="Avatar" className="avatar" />

```